### PR TITLE
create welcome blogpost PH-FR in EN and FR, closes #1161 and #1184

### LINKED ISSUE #1161: introducing the PH en Français - blogpost in French
As we are witnessing the successful  launch of the Programming Historian en français 🚀, it would be great to write a blogpost en français introducing the initiative to our French-speaking readers around the world, asking them to contribute with French  tutorials and/or translating existing lessons. Feel free also to use the twitter in order to post tweets  en français - we are also having sporadically tweets in Spanish.

@spapastamkou @mpuren @fdlaramee are you ready to rock?

### PR DESCRIPTION
**Pull request by spapastamkou**
I asked reviews from the @programminghistorian/english-team for the EN text and from the @programminghistorian/french-team for the FR text. 
Also, I have a small doubt about the image, that it might be too heavy. Do you think you can take a look, @mdlincoln and let me know if ok? It is [here](https://github.com/programminghistorian/jekyll/tree/welcome-ph-fr-blogposts/images/welcome-ph-fr). Thank you

### CODE REVIEW

**Review by drjwbaker — DISMISSED** [review]
EN post is great. Thanks for this. If I was being picky, I'd lightly edit, but I think leaving as it is better captures the style I think you are going for in the FR original! On the image, do we have permission to republish it CC-BY? If not, we may want to mark the image as "© Mundaneum".

**Review by mpuren — CHANGES_REQUESTED** [review]
Thanks a lot for this blogpost, it is perfect. One minor correction in the french version : "mais aussi d'autres chercheur(e)s en sciences humaines et sociales" -> "mais aussi d'autres chercheur(se)s en sciences humaines et sociales"

**Review by mpuren — DISMISSED** [review]

**Comment by spapastamkou** [comment]
Thank you @drjwbaker and @mpuren. 
@drjwbaker, you may edit if you think the text will be better:)
@mpuren, I put chercheuses instead of chercheures and split to avoid parenthesis AND "s" in the middle:)

**Comment by mdlincoln** [comment]
@spapastamkou Can you please reduce the image to maybe 800px width? That's the max size it will display on the site, that will reduce the file size a lot.

File: _posts/2019-08-04-bienvenue-ph-fr.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Bienvenue au Programming Historian en français!
+authors:
+- Sofia Papastamkou
+layout: post
+categories: posts
+---
+
+<p><figure><img src="/images/welcome-ph-fr.png" /><figcaption>Schéma de Paul Otlet: documentation et télécommunication, <i>Mundaneum, Centre d'archives de la Fédération Wallonie-Bruxelles, Mons, Belgique. © Mundaneum</i></figcaption></figure></p>
```
Comment (mdlincoln) [inline _posts/2019-08-04-bienvenue-ph-fr.md]: You have a broken image link - this should be `images/welcome-ph-fr/welcome-ph-fr.png`

**Review by mdlincoln — CHANGES_REQUESTED** [review]

**Review by mdlincoln — APPROVED** [review]

**Comment by spapastamkou** [comment]
oh my God! I just tweeted the EN version of the post and date appears as 4 August:( It was 4 April, I surely reversed the day and month in the format. Sorry. If I create a new branch and rename the files, will you hate me, @mdlincoln ?

**Pull request merged**